impostor.kvp

```diff
@@ -0,0 +1,88 @@
+Meta.DisplayName=Impostor
+Meta.Description=Impostor Dedicated Server
+Meta.OS=Windows, Linux
+Meta.Arch=x86_64
+Meta.Author=Greelan
+Meta.URL=https://github.com/Impostor/Impostor
+Meta.DisplayImageSource=url:https://github.com/Impostor/Impostor/raw/master/docs/images/logo_458.png
+Meta.EndpointURIFormat=
+Meta.ConfigManifest=impostorconfig.json
+Meta.MetaConfigManifest=impostormetaconfig.json
+Meta.ConfigRoot=impostor.kvp
+Meta.MinAMPVersion=2.4.0.10
+Meta.SpecificDockerImage=
+Meta.DockerRequired=False
+Meta.ContainerPolicy=Supported
+Meta.ContainerPolicyReason=
+Meta.Prerequsites=[]
+Meta.ExtraContainerPackages=[]
+Meta.ConfigReleaseState=NotSpecified
+Meta.NoCommercialUsage=False
+App.DisplayName=Impostor
+App.RootDir=./impostor/
+App.BaseDirectory=./impostor/
+App.ExecutableWin=Impostor.Server.exe
+App.ExecutableLinux=Impostor.Server
+App.WorkingDir=
+App.LinuxCommandLineArgs=
+App.WindowsCommandLineArgs=
+App.CommandLineArgs={{$PlatformArgs}} {{$FormattedArgs}}
+App.UseLinuxIOREDIR=False
+App.AppSettings={}
+App.EnvironmentVariables={"DOTNET_ROOT":"{{$FullBaseDir}}dotnet"}
+App.CommandLineParameterFormat=-{0} {1}
+App.CommandLineParameterDelimiter= 
+App.ExitMethod=OS_CLOSE
+App.ExitTimeout=30
+App.ExitString=quit
+App.ExitFile=app_exit.lck
+App.HasWriteableConsole=True
+App.HasReadableConsole=True
+App.SupportsLiveSettingsChanges=False
+App.LiveSettingChangeCommandFormat=set {0} "{1}"
+App.ApplicationIPBinding=0.0.0.0
+App.Ports=[{"Protocol":"UDP","Port":22023,"Ref":"GamePort","Name":"Game Port","Description":"Port for game traffic"}]
+App.AdminPortRef=0
+App.PrimaryApplicationPortRef=GamePort
+App.UniversalSleepApplicationUDPPortRef=GamePort
+App.UniversalSleepSteamQueryPortRef=0
+App.MaxUsers=100
+App.UseRandomAdminPassword=False
+App.RemoteAdminPassword=Password123
+App.AdminMethod=STDIO
+App.AdminLoginTransform=None
+App.RCONConnectDelaySeconds=5
+App.RCONConnectRetrySeconds=5
+App.RCONHeartbeatMinutes=0
+App.RCONHeartbeatCommand=ping
+App.TelnetLoginFormat=login {0}
+App.UpdateSources=@IncludeJson[impostorupdates.json]
+App.Compatibility=None
+App.SteamUpdateAnonymousLogin=False
+App.SteamForceLoginPrompt=False
+App.RapidStartup=False
+App.MonitorChildProcess=False
+App.MonitorChildProcessWaitMs=100
+App.MonitorChildProcessName=
+App.SupportsUniversalSleep=True
```

Comment:
It doesn't seem to support sleep mode. It doesn't wake up on connection requests.

Reply:
It does on Linux. Seems the Windows bug strikes again. For that reason, and because the instance crashes if the Stop button is clicked while the server is sleeping, I've disabled Sleep.

Reply:
*Facepalm emoji*